client/app/pages/alerts/AlertsList.jsx

```diff
@@ -94,7 +94,7 @@ class AlertsList extends React.Component {
                   totalCount={controller.totalItemsCount}
                   itemsPerPage={controller.itemsPerPage}
                   page={controller.page}
-                  onChange={page => controller.updatePagination({ page })}
+                  onChange={(page) => controller.updatePagination({ page })}
```

Comment:
@gabrieldutra @kravets-levko if you format locally this file, does it change it like restyled try to do here?

Reply:
No, the local format is differing from restyled, something is wrong with it (I was waiting to see if it was only my PR or if there was others)